src/js/electron/userTasks.ts

```diff
@@ -6,21 +6,29 @@ export default function(app: any) {
     app.setUserTasks([
       {
         program: process.execPath,
-        arguments: getArguments(),
+        arguments: getArguments("--new-window"),
         iconPath: process.execPath,
         iconIndex: 0,
         title: "New Window",
         description: "Create a new window"
+      },
+      {
+        program: process.execPath,
+        arguments: getArguments("--move-to-current-display"),
+        iconPath: process.execPath,
+        iconIndex: 0,
+        title: "Move Brim to Current Display",
+        description: "Move brim windows to the current display."
```

Comment:
nit: capitalize Brim like you did for `title`

Reply:
Oops yes, thanks!